<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>985</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="Infos_connexion">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="Nom_utilisateur"/>
       </item>
       <item>
        <widget class="QRadioButton" name="Connexion">
         <property name="text">
          <string>Connecté</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Infos_musique">
       <item>
        <widget class="QLabel" name="Image">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="Liste"/>
       </item>
       <item>
        <widget class="QListView" name="Morceaux"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Infos_temps_lecture">
       <item>
        <widget class="QLabel" name="Temps_restant">
         <property name="text">
          <string>--:--:--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Progression">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Temps_lu">
         <property name="text">
          <string>--:--:--</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Commandes">
       <item>
        <widget class="QPushButton" name="Rewind">
         <property name="text">
          <string>rewind</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Back">
         <property name="text">
          <string>back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Play_pause">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Next">
         <property name="text">
          <string>next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Foward">
         <property name="text">
          <string>foward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Mute">
         <property name="text">
          <string>Mute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Volume">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModes">
    <property name="title">
     <string>Modes</string>
    </property>
    <addaction name="actionMorceaux"/>
    <addaction name="separator"/>
    <addaction name="actionRadio"/>
   </widget>
   <widget class="QMenu" name="menuLangues">
    <property name="title">
     <string>Langues</string>
    </property>
    <addaction name="actionAnglais"/>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
    <addaction name="separator"/>
    <addaction name="actionFran_ais"/>
    <addaction name="separator"/>
    <addaction name="actionOccitan"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuModes"/>
   <addaction name="menuLangues"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMorceaux">
   <property name="text">
    <string>Morceaux</string>
   </property>
  </action>
  <action name="actionRadio">
   <property name="text">
    <string>Radio</string>
   </property>
  </action>
  <action name="actionAnglais">
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFran_ais">
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionOccitan">
   <property name="text">
    <string>Occitan</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
